mleap-tensorflow/src/test/scala/ml/combust/mleap/tensorflow/converter/MleapConverterSpec.scala

```diff
@@ -0,0 +1,192 @@
+      val ml_tensor_converted = TensorflowConverter.convert(tf_tensor_converted, TensorType.Float())
+      assert(ml_tensor.dimensions == ml_tensor_converted.dimensions)
+      assert(ml_tensor.base == ml_tensor_converted.base)
+      assert(ml_tensor.size == ml_tensor.size)
+      assert(array sameElements ml_tensor.toArray)
+      assert(ml_tensor.toArray sameElements ml_tensor_converted.toArray)
+      assert(
+        Helper.scalars(shape).map(indices => tf_tensor_converted.getObject(indices.map(_.toLong): _*))
+          sameElements Helper.scalars(shape).map(indices => ml_tensor(indices: _*))
+      )
+
+    }
+
+    it("double nd dense tensors") {
+      val random = new java.security.SecureRandom
+      val shape = Seq(3, 3, 3, 3, 3, 3, 3)
+      val size = shape.product
+      val array = Array.fill(size) {
+        random.nextDouble()
+      }
+      val ml_tensor = Tensor.create(array, dimensions = shape)
+      val tf_tensor_converted = MleapConverter.convert(ml_tensor, TensorType.Double()).asInstanceOf[TFloat64]
+      val ml_tensor_converted = TensorflowConverter.convert(tf_tensor_converted, TensorType.Double())
+      assert(ml_tensor.dimensions == ml_tensor_converted.dimensions)
+      assert(ml_tensor.base == ml_tensor_converted.base)
+      assert(ml_tensor.size == ml_tensor.size)
+      assert(array sameElements ml_tensor.toArray)
+      assert(ml_tensor.toArray sameElements ml_tensor_converted.toArray)
+      assert(
+        Helper.scalars(shape).map(indices => tf_tensor_converted.getObject(indices.map(_.toLong): _*))
+          sameElements Helper.scalars(shape).map(indices => ml_tensor(indices: _*))
+      )
+    }
+    it("byte nd dense tensors") {
+      val random = new java.security.SecureRandom
+      val shape = Seq(3, 3, 3, 3, 3, 3, 3)
+      val size = shape.product
+      val array = Array.fill[Byte](size) {
+        0
+      }
+      random.nextBytes(array)
+      val ml_tensor = Tensor.create(array, dimensions = shape)
+      val tf_tensor_converted = MleapConverter.convert(ml_tensor, TensorType.Byte()).asInstanceOf[TUint8]
+      val ml_tensor_converted = TensorflowConverter.convert(tf_tensor_converted, TensorType.Byte())
+      assert(ml_tensor.dimensions == ml_tensor_converted.dimensions)
+      assert(ml_tensor.base == ml_tensor_converted.base)
+      assert(ml_tensor.size == ml_tensor.size)
+      assert(array sameElements ml_tensor.toArray)
+      assert(ml_tensor.toArray sameElements ml_tensor_converted.toArray)
+      assert(
+        Helper.scalars(shape).map(indices => tf_tensor_converted.getObject(indices.map(_.toLong): _*))
+          sameElements Helper.scalars(shape).map(indices => ml_tensor(indices: _*))
+      )
+    }
+
+    it("byte string nd dense tensors") {
+      val random = new java.security.SecureRandom
+      //      val shape = Seq(3, 3, 3, 3, 3, 3, 3)
+      val shape = Seq(3, 3)
```

Comment:
why this was needed? (i'm mostly just curious and don't really expect any code changes)

Reply:
I forget to change back to val shape = Seq(3, 3, 3, 3, 3, 3, 3), I use simpler shape for debug